proof/invariant-abstract/DetSchedSchedule_AI.thy

```diff
@@ -5178,10 +5182,17 @@ lemma sched_context_resume_valid_sched_except_blocked:
    sched_context_resume scp
    \<lbrace>\<lambda>_. valid_sched_except_blocked\<rbrace>"
   apply (wpsimp wp: postpone_valid_sched_except_blocked is_schedulable_wp' get_tcb_queue_wp thread_get_wp'
-              simp: sched_context_resume_def refill_sufficient_def get_refills_def refill_ready_def)
-  apply (clarsimp simp: valid_sched_def sc_at_pred_n_def get_tcb_ko_at obj_at_def)
-  apply (frule (1) heap_refs_retract_atD, clarsimp simp: vs_all_heap_simps)
-  apply (prop_tac "\<not> released_sc_tcb_at tp s", clarsimp simp: vs_all_heap_simps refill_ready_def refill_sufficient_def)
+              simp: sched_context_resume_def get_refills_def)
+  apply (clarsimp simp: obj_at_def)
+  apply (prop_tac "heap_ref_eq y scp (sc_tcbs_of s)")
```

Comment:
Is it at all nicer to `rename_tac` this `y`? I can see there is a `tp` further down, so maybe this is a bit of a confusing one